diffkemp/simpll/SourceCodeUtils.cpp

```diff
@@ -50,6 +51,8 @@ std::unordered_map<std::string, MacroElement> getAllMacrosOnLine(
             // from the left, record all such substrings and test them using the
             // macro map provided in the function arguments.
             std::string macroBody = Entry.second.body.str();
+            expandCompositeMacroNames(Entry.second.params, Entry.second.args,
+                    macroBody);
```

Comment:
Better alignment of parameters (applies also to all function call/definitions below):

    expandCompositeMacroNames(Entry.second.params, Entry.second.args,
                              macroBody);